Sources/Plasma/PubUtilLib/plInputCore/plAvatarInputInterface.h

```diff
@@ -229,7 +232,7 @@ class plAvatarInputInterface : public plInputInterface
         uint32_t        GetPriorityLevel() const override { return kAvatarInputPriority; }
         uint32_t        GetCurrentCursorID() const override { return fCurrentCursor; }
         float           GetCurrentCursorOpacity() const override { return fCursorOpacity; }
-        const char*         GetInputMapName() { return fInputMap ? fInputMap->GetName() : ""; }
+        ST::string      GetInputMapName() { return fInputMap ? fInputMap->GetName() : ST::string(); }
```

Comment:
```suggestion
        ST::string      GetInputMapName() { return fInputMap ? fInputMap->GetName() : {}; }
```

Reply:
That doesn't work unfortunately. CLion/clangd says "Initializer list cannot be used on the right hand side of operator ':'", and MSVC seems to agree:

```
.../plAvatarInputInterface.h(235): error C2059: syntax error: '{'
.../plAvatarInputInterface.h(235): error C2143: syntax error: missing ';' before '{'
```

Reply:
😞